pyproject.toml

```
@@ -1,6 +1,71 @@
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "jupyter_enterprise_gateway"
+description = "A web server for spawning and communicating with remote Jupyter kernels"
+license = { file = "LICENSE.md" }
+keywords = ["Interactive","Interpreter","Kernel", "Web", "Cloud"]
+classifiers = [
+  "Intended Audience :: Developers", "Intended Audience :: Science/Research",
+  "Intended Audience :: System Administrators",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10"
+]
+requires-python = ">=3.7"
+dependencies = [
+  "docker>=3.5.0",
+  "future",
+  "jinja2>=3.1",
+  "jupyter_client~=6.1",
+  "jupyter_core>=4.6.0",
+  "kubernetes>=4.0.0",
+  "jupyter_server>=1.2",
+  "paramiko>=2.1.2",
+  "pexpect>=4.2.0",
+  "pycryptodomex>=3.9.7",
+  "pyzmq>=17.0.0",
+  "requests~=2.7",
```

In this case, I think this "cap" can be removed since `requests` hasn't had a major version since 2013.  As with others, we can simply take our chances.
```suggestion
  "requests>=2.7",
```